src/Type/Definition/ResolveInfo.php

```diff
@@ -187,19 +184,16 @@ public function getFieldSelection($depth = 0)
         return $fields;
     }
 
-    public function lookAhead() : QueryPlan
+    public function lookAhead(string ...$options) : QueryPlan
```

Comment:
One last note: please change it to just `$options` array vs destructuring to make it consistent with other usages of options in this lib